resources/datadog-integrations/software.rb.erb

```diff
@@ -4,9 +4,19 @@ require 'json'
 name "dd-check-<%= name %>-software"
 default_version "0.0.1"
 
-require_relative "<%= PROJECT_DIR %>/resources/datadog-integrations/validate_manifest"
+if windows?
+  require_relative "../../resources/datadog-integrations/validate_manifest"
+else 
+  require_relative "<%= PROJECT_DIR %>/resources/datadog-integrations/validate_manifest"
+end
+
 integration_source_folder = '/<%= integrations_repo %>/<%= name %>'
-integration_agent_folder = "#{install_dir}/integrations/<%= name %>"
+
+if windows?
+  integration_agent_folder = install_dir
```

Comment:
There are a small predictable number of files outside the install_dir (I can only think of two), so there's no reason not to constrain the installation. We might also be able to do this on linux, but that's a discussion for a different day.